integrations/google-tag-manager.md

```diff
@@ -3,6 +3,22 @@
 > **Notice of usage:** Google Tag Manager (GTM) and Google Analytics 4 (GA4) are third party services and we provide this integration as is.
 We support a set of custom events and Data Layer variables for use with GTM and GA4, however we have no control over what happens to them and how they are used. The basic setup examples we provide have been tested and verified to work with the Mews Booking Engine, however if you need a more complex setup then we cannot provide the support to do so and we recommend to ask a specialist to set it up and test it for you.
 
+
```

Comment:
Copy/paste edited version here, from the integrations home page.

Reply:
Done, thanks for corrections 👍